.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.8'
+
+services: 
+  development:
+    build:
+      context: .
+      dockerfile: Dockerfile
+  app:
+    depends_on:
+      - app_db
+    image: ghcr.io/wp-graphql/wp-graphql:1.16.0-wp6.3-php8.2
```

Comment:
We'd ideally want this to be latest and not hardcoded

Reply:
I'll check, not sure if we have those tags.

Reply:
It's not the end of the world if not, were already doing a manual string replace when we need to, but it would be nice to reduce the maintenance

Reply:
the closest we have is `[latest-wp6.3-php8.2](https://github.com/orgs/wp-graphql/packages/container/wp-graphql-testing/123900207?tag=latest-wp6.3-php8.2)` That atleast covers WPGraphQL which is the important bit, even if it doesn't include the latest WP or PHP. Should we create an issue to have a more general `latest` tag that covers all bits?